server/cli/accounting.go

```diff
@@ -137,3 +137,18 @@ func runSetAcct(cmd *cobra.Command, args []string) {
 }
 
 // TODO:(bram) Add inline json for setting
+
+var acctCmds = []*cobra.Command{
+	getAcctCmd,
+	lsAcctsCmd,
+	rmAcctCmd,
+	setAcctCmd,
+}
+
+var acctCmd = &cobra.Command{
+	Use: "account",
```

Comment:
I wonder if we should be explicit and call this accounting. "account" is maybe a little misleading or confusing.